catalog/templates/column.html

```diff
@@ -48,12 +49,34 @@
 
    .. tab:: {"tab_name": "Boundaries by identifier in another table"}
 
+      Update the boundaries in another table matching the table's ID column.
+
+      .. code-block:: postgresql
+
+          WITH data AS (
+            SELECT cartodb_id AS id
+            FROM <table_to_update>
+          ),
+          meta AS (
+            SELECT OBS_GetMeta(ST_MakeEnvelope(-179, 89, 179, -89, 4326),
+              '[{"geom_id": "{{ col.id }}"}]'
+            ) meta
+          )
+          UPDATE <table_to_update> SET
+                geomref = data->0->>'geomref',
+                the_geom = (data->0->>'value')::Geometry,
+                the_geom_webmercator = ST_Transform((data->0->>'value')::Geometry, 3857)
+          FROM OBS_GetData(
+                (SELECT Array_agg(id) FROM data),
+                (SELECT meta FROM meta)) obs_data
+          WHERE <table_to_update>.cartodb_id = obs_data.id
+
       Obtain boundaries matched to another table's ID column.
 
       .. code-block:: postgresql
 
           WITH ids AS (
-          SELECT ARRAY_AGG(<id_col>) ids
+          SELECT ARRAY_AGG(cartodb_id) ids
```

Comment:
Uhm, this is actually not `cartodb_id`, but rather, the geoid of the geometry. With cartodb_ids, the query doesn't work, as it expects a string array, not an integer array.

Reply:
You are right, the column must be `geo_id` instead of `cartodb_id`